### DISCUSSION TITLE
how to rebuild the api project for Serenity.Script

### DISCUSSION THREAD

**Discussion by kuyu7766** [serenity-is/Serenity#6945] [Help]
In serenity version 6.7.1  I want to change the language to 'zh-CN' on datepicker.  So I have to change the code line "`let culture = ($('html').attr('lang') || 'zh-CN');`" in  file "dateeditor.ts" which is in   "Serenity-master\Serenity-master\packages\corelib\src\ui\editors" 。
If I change this code  how do I rebuild the project Serenity.Script?  now the project of  Serenity.Script have error about 
/// `<reference types="jquery" />`
/// `<reference types="jquery.validation" />`
show error message  Error TS2688 (TS) could not find the type definition file for "jquery".

**Comment by VictorTomaili** [comment]
You don't need to change the source. As you can see, the language is retrieved from `html` tag `lang` attribute. 
It should be like `<html lang='zh-CN'>`

**Reply by kuyu7766** [reply]
Thank you for your answer.But I noticed  th function jQueryDatepickerInitialization in the Serenity.CoreLib (which is in the project Serenity.Script) has a statement `var culture = ($('html').attr('lang') || 'en').toLowerCase();`. Because the "CurrentUICulture.Name" is uppercase,so the datepicker can't find the right language resource.  That is why i want to fix the code and get rid of the `.toLowerCase()` and rebuild the project Serenity.Script.

**Reply by VictorTomaili** [reply]
You can use `($ as any).datepicker.setDefaults(($ as any).datepicker.regional[($('html').attr('lang') || 'en')]);` in your scriptinit.ts file
